src/detectors/FHCAL/FHCAL.cc

```diff
@@ -89,13 +89,14 @@ extern "C" {
         app   // TODO: Remove me once fixed
       ));
 
-      double side_length=18.89 * dd4hep::mm;
+      // define the distance between neighbors in terms of the largest possible distance between subcell hits
+      double side_length=31 * dd4hep::mm;
```

Comment:
Would it make sense to access the `HcalEndcapPInsertCellSizeLGRight` from the geometry? We'd need to provide a fallback value when it's not defined.

Reply:
that makes sense, I'll look into how to access that information

Reply:
thanks!  I'll try this.

Reply:
done